lighthouse-cli/performance-experiment/report/scripts/perf-x.js

```diff
@@ -31,7 +31,8 @@ class ConfigPanel {
     this._messageField = this._configPanel.querySelector('.js-message');
     this._urlBlockingList = this._configPanel.querySelector('.js-url-blocking-patterns');
     this._urlBlockingStatus = {};
-    this._reportId = new URL(window.location).searchParams.get('id');
+    const match = location.search.match(/[?&]id=([^&]*)/);
```

Comment:
I use a polyfill in Viewer for URLSearchParams: https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-viewer/package.json#L33

Could you use that here?